capstone/capdb/models.py

```diff
@@ -783,18 +778,16 @@ def create_or_update_metadata(self, update_existing=True, save_self=True):
         return case_metadata
 
 
-class Citation(AutoSlugMixin, models.Model):
+class Citation(models.Model):
     type = models.CharField(max_length=100,
                             choices=(("official", "official"), ("parallel", "parallel")))
     cite = models.CharField(max_length=10000, db_index=True)
     duplicative = models.BooleanField(default=False)
-    slug = models.SlugField(max_length=255, unique=True)
+    normalized_cite = models.SlugField(max_length=255, null=True, db_index=True)
```

Comment:
We have cites longer than 255. I think they're all data errors, but maybe increase the size of this field for now.

Reply:
Oh also I think it's a good idea to automatically set normalized_cite from cite on save, if `self.tracker.has_changed('cite')`